app/src/main/res/layout/fragment_splash.xml

```diff
@@ -1,7 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
-<TextView xmlns:android="http://schemas.android.com/apk/res/android"
-    style="@style/TextAppearance.AppCompat.Headline"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Assets used as backgrounds should have the `bg_` prefix, so `bg_first_onboarding_page`

Assets like `move_logo` can be considered icons and we prefix them with `ic_`

`scooter_image` is in a weird place when it comes to naming, so it's up to you. Personally I'd go with `bg_splash_scooter` or `bg_splash`.

`splash_background_image` should also respect the prefix naming convention. Could be `ic_`, could be `bg_`. And as a note, in asset names we prefer avoiding words like "image", "video", "gif".